scripts/test.sh

```diff
@@ -3,10 +3,6 @@ set -e
 
 service mysql start
 
-cat example/datasets/popularize_churn.sql | mysql -uroot -proot
```

Comment:
`sql/db_test.go` is responsible for data population.

Reply:
I'm not sure `db_test.go` can always run before `main_test` since `main_test` depend on these sample data.

Reply:
You are right. I forgot the tests in `cmd/` also needs populated data. My bad.